http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1613,6 +1613,281 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
+  "/proxy/{googleProject}/{runtimeName}/yarn":
+    get:
+      summary: YARN Resource Manager
+      description: >
+        Only available for Dataproc runtimes with Component Gateway enabled.
```

Comment:
On the UX, UI, User Ed and Front-line realms, we've been trying to deprecate the term `runtime` in favour of `cloud compute` (or just `compute`) where a `cloud environment` is composed of `application config`, `cloud compute` and optionally `persistent disk`. I wonder if we should try to do the same on the backend, at least on the user-facing portions. Having a consistent terminology makes communication and onboarding easier.

Reply:
I hear you but all the Swagger routes/models/etc are named `runtimes` so I think this is consistent from a Swagger perspective..